proposals/0002-ledger-car.md

```diff
@@ -0,0 +1,274 @@
+
+The reference operator (`&`) marks the locations where subtrees of the structure are split out into separate blobs. (specifically, every `Entry` and `Transaction` struct instance)
+The original position of the split out object is replaced with a CID link.
+
+Data is encoded into blobs by encoding the wire representation data model (as indicated by the schema) with [DAG-CBOR] codec.
+
+  [DAG-CBOR]: https://ipld.io/specs/codecs/dag-cbor/spec/
+
+### Multicodec IDs
+
+The type of each blob is defined by the outermost structure and identified by the multicodec ID.
+
+For the Ledger-CAR types, the IDs are defined as follows.
+- `Transaction` (legacy or v0 transaction format): `0x5B00`
+- `Block`: `0x5BCB`
+- `Entry`: `0x5BCE`
+
+### CAR blob order
+
+Blobs stored in the Ledger-CAR file follow a strict order.
+
+For each block, the order of blobs matches that of depth-first traversal over the aforementioned IPLD representation.
+
+In the case of CARs storing multiple blocks, blocks should be ordered by block number.
+
+**Listing 1: Example block IPLD data**
+
+```
+Block {
+  slot: 42,
+  entries: [
+    Entry {
+      numHashes: 100,
+      hash: 'foo',
+      txs: [
+        Transaction: 'tx1',
+        Transaction: 'tx2',
+      ]
+    }
+    Entry {
+      numHashes: 101,
+      hash: 'bar',
+      txs: [
+        Transaction: 'tx3',
+      ]
+    }
+  ]
+}
+```
+
+**Listing 2: Example block in CAR blob order**
+
+The block in _listing 1_ corresponds to the following ordered vector of blobs.
+
+1. ```
+   Transaction: 'tx1'
+   ```
+2. ```
+   Transaction: 'tx2'
+   ```
+3. ```
+   Entry {
+     numHashes: 100,
+     hash: 'foo',
+     txs: [ CID_1, CID_2 ]
+   }
+   ```
+4. ```
+   Transaction: 'tx3'
+   ```
+5. ```
+   Entry {
+     numHashes: 101,
+     hash: 'bar',
+     txs: [ CID_4 ]
+   }
+   ```
+6. ```
+   Block {
+     slot: 42,
+     entries: [ CID_3, CID_5 ]
+   }
+   ```
+
+### CID Format
+
+CIDs in Ledger-CAR must use the following parameters:
+- multibase prefix: `b` (base32)
+- multihash type: `0x12` (sha2-256)
+- multihash length: `0x20` (32 bytes)
+
+## Motivation
+
+Transparency and verifiability of transaction data are fundamental security properties of public blockchains.
+In practice, these directly depend on public availability of block data.
+
+Existing methods for storing distributing historical block data bear various shortcomings, such as the use of trusted peers or private networks.
+
+This warrants the creation of file format designed to support ledger archival and public distribution.
+This new format should be generic/neutral (to support a variety of storage backends) and simple to implement (read: based on existing standards).
+Additionally, any node operator that owns an existing local copy of ledger data should be able to independently convert their data into this new format to support public distribution efforts.
+
+## Alternatives Considered
```

Comment:
Should also add design considerations sections. e.g. explaining the usage of CARv1 with CIDs (which add considerable overhead), instead of just a flat archive like `.tar`